### PR TITLE
Lesson8

### PR DESCRIPTION
**Pull request by oleksandrtest**

### CODE REVIEW

File: lesson7/src/Main.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import java.util.Arrays;
+
+public class Main {
+
+    public static void main(String[] args){
+
+        var nums = new int[]{14, 231, 2, 3214, 2, 434};
+        var sum = 0;
+        var prev = 0;
+
```
Comment (shaitanaleksander) [inline lesson7/src/Main.java:10]: you can solve this task without these two variable

File: lesson8/src/one/Triangle.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package one;
+
+import one.Shape;
+
+public class Triangle implements Shape {
+
+    private final int a = 34;
+    private final int b = 56;
```
Comment (shaitanaleksander) [inline lesson8/src/one/Triangle.java:8]: I know it is hard to get the correct name for this kind of variable but you need to do better than these

File: lesson8/src/one/Triangle.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+
+    @Override
+    public int area() {
+
```
Comment (shaitanaleksander) [inline lesson8/src/one/Triangle.java:14]: formatting, empty space

File: lesson8/src/two/Barrier.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package two;
+
```
Comment (shaitanaleksander) [inline lesson8/src/two/Barrier.java:2]: what this name means, need to be renamed

**Review by shaitanaleksander — APPROVED** [review]

File: lesson7/src/Main.java (hunk 1 above)
Comment (oleksandrtest) [reply lesson7/src/Main.java:10]: Це завдання вже перевірено, просто Іван його забув заапрувити!

**Pull request merged**